### PR TITLE
Kb for glassfish deployer

### PR DESCRIPTION
**Pull request by itsmeroy2012**
## Documentation for glassfish_deployer

### CODE REVIEW

**Comment by wchen-r7** [comment]
That's very shocking I didn't add module documentation for GlassFish, thanks @itsmeroy2012

**Comment by itsmeroy2012** [comment]
The pleasure is all mine @wchen-r7

**Comment by h00die** [comment]
for your commit messages, no need to do 'update xyz', just put a short change item 'added example exploitation', 'fixed spelling' etc. no need to revision control w/ numbers

File: documentation/modules/exploit/multi/http/glassfish_deployer.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## Description
+
+This module logs in to an GlassFish Server (Open Source or Commercial) using various methods (such as authentication bypass, default credentials, or user-supplied login), and deploys a malicious war file in order to get remote code execution. It has been tested on Glassfish 2.x, 3.0, 4.0 and Sun Java System Application Server 9.x. Newer GlassFish versions do not allow remote access (Secure Admin) by default, but is required for exploitation.
```
Comment (h00die) [inline documentation/modules/exploit/multi/http/glassfish_deployer.md]: an, or a?

**Comment by h00die** [comment]
I was going to comment that its always nice to say the version of software thats being exploited in the docs, however, in the code block it calls out the version, so no need in this case.

These docs look good, gets my sign off once the slight change above is fixed

**Comment by wchen-r7** [comment]
# Release Notes

This adds module documentation for exploit/multi/http/glassfish_deployer.

**Pull request merged**